PokedexMentoria/Home/HomeViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  HomeViewController.swift
+//  PokedexMentoria
+//
+//  Created by Matheus Rodrigues Araujo on 23/12/22.
+//
+
+import UIKit
+
+protocol HomeCoordinatorDelegate: AnyObject {
+    
+}
+
+final class HomeViewController: UIViewController {
+    
+    weak var homeCoordinatorDelegate: HomeCoordinatorDelegate?
+    
+    // MARK: - ViewCode
+    
+    private lazy var mainStack: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        PokeballCard.Style.allCases.forEach { style in
+            let card = PokeballCard(style: style)
+            card.delegate = self
+            stackView.addArrangedSubview(card)
+        }
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = .defaultSpacement
+        return stackView
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        makeLayoutHierarch()
+    }
+    
+    func makeLayoutHierarch() {
```

Comment:
se for ser acessado só por métodos da própria classe, pode ser private

Reply:
corrigido